### PR TITLE
fix assemble_crop

### PR DESCRIPTION
**Pull request by a-arefina**

### CODE REVIEW

**Review by nikita-klsh — DISMISSED** [review]

**Review by anton-br — DISMISSED** [review]

**Comment by nikita-klsh** [comment]
Please specify in the docs of `assemble_crops` how the aggregation is being performed

File: seismicpro/src/seismic_batch.py [hunk 1]
```diff
@@ -725,7 +725,9 @@ def __crop(self, index, src, coords, shape, pad_zeros, dst=None):
     @apply_to_each_component
```
Comment (anton-br) [inline seismicpro/src/seismic_batch.py:725]: delete this decorator if you don't expect src or dst to be an array of components

File: seismicpro/src/seismic_batch.py [hunk 2]
```diff
@@ -725,7 +725,9 @@ def __crop(self, index, src, coords, shape, pad_zeros, dst=None):
...
     def assemble_crops(self, index, src, dst, fill_value=0.0):
         """
```
Comment (anton-br) [inline seismicpro/src/seismic_batch.py:727]: add raises to the docs

Comment (anton-br) [reply seismicpro/src/seismic_batch.py:727]: and note that given data can't be SegyFilesIndex

**Review by anton-br — APPROVED** [review]

**Review by nikita-klsh — APPROVED** [review]

**Review by dpodvyaznikov — APPROVED** [review]

**Pull request merged**